packages/wp-now/src/wp-playground-wordpress/tests/get-plugin-file.spec.ts

```diff
@@ -0,0 +1,26 @@
+import path from 'path';
```

Comment:
Can we also update the integration tests to verify the plugin (and theme) is activated the first time around?

Reply:
Added the tests here:
- https://github.com/WordPress/wordpress-playground/pull/351/commits/5c1f71d26104aa5e67b8f80c9aa9c05217d70f4f
- https://github.com/WordPress/wordpress-playground/pull/351/commits/4438ed0817bdd12b26f2f942bc6904f5bc230fe4
- https://github.com/WordPress/wordpress-playground/pull/351/commits/bda1d0d55629d95e8cb77caa17924fe0894f0340